apps/mobile/src/components/add-wallet/add-wallet-sheet.layout.tsx

```diff
@@ -128,12 +140,13 @@ export function AddWalletSheetLayout({
               ? null
               : Object.entries(getUnavailableFeatures(theme)).map(featureEntry => {
                   const [featureKey, feature] = featureEntry;
+                  function fallbackOnPress() {
+                    onOpenNotificationsSheet({ title: feature.title, id: featureKey });
+                  }
                   return (
                     <AddWalletListItem
                       key={featureKey}
-                      onPress={() =>
-                        onOpenNotificationsSheet({ title: feature.title, id: featureKey })
-                      }
+                      onPress={feature.onPress ?? fallbackOnPress}
```

Comment:
Maybe we can keep the function anonymous here as otherwise the `.map` will create `fallbackOnPress` even if it has `feature.onPress`